src/compute-engine/boxed-expression/match.ts

```diff
@@ -339,39 +339,41 @@ function matchArguments(
 
     if (argName !== null) {
       if (argName.startsWith('__')) {
-        // Match 0 or more expressions (__) or 1 or more (___)
-        let j = 0; // Index in subject
+        // Match 1 or more expressions (__) or 0 or more (___)
+        /** Qty. of operands consumed (of those remaining to be matched). */
+        let j = 0;
```

Comment:
Changes here overridden by later commit (68f23c3).
Notably, `j` instead referencing 'qty. of operands consumed', is preserved.